packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -1523,7 +1531,13 @@ export default class TradeStore extends BaseStore {
             });
         }
         if (req.active_symbols) {
-            return WS.activeSymbols('brief');
+            /** returning null here to avoid exrtra active_symbol request
+             * from server since this is getting called in chart init and
+             * active_symbols is already being fetched in trade.jsx init
+             */
+            return null;
```

Comment:
wsSendRequest is passed into SmartCharts so that it can request active_symbols (via deriv-app here), so active symbols (e.g. this.active_symbols) have to be returned here instead of null. Otherwise, SmartCharts won't get active symbols.

Reply:
Maybe we can `return this.active_symbols` instead? 🤔

Reply:
Done 🙏